sdks/python/apache_beam/transforms/external.py

```diff
@@ -498,11 +502,24 @@ def expand(self, pvalueish):
               spec=beam_runner_api_pb2.FunctionSpec(
                   urn=common_urns.primitives.IMPULSE.urn),
               outputs={'out': transform_proto.inputs[tag]}))
+    output_coder = None
+    if self._type_hints.output_types:
+      if self._type_hints.output_types[0]:
+        output_coder = dict((str(k), context.coder_id_from_element_type(v))
+                            for k,
```

Comment:
You can do `for (k, v) in ...` to make yapf happy. (Same below.)

Reply:
Thanks. Better looking with (k, v) syntax.